### DISCUSSION TITLE
Guide for medical prototype.

### DISCUSSION THREAD

**Discussion by NomadsReach** [acemod/ACE-Anvil#114] [Feature Requests]
### I have created a guide for the current medical system and would like to propose its inclusion in the team’s documentation. Please let me know if this would be of interest.

[Direct Link](https://www.canva.com/design/DAGRzLkF9b4/aYYVPpP9ElVTMHwP6iW3ig/view?utm_content=DAGRzLkF9b4&utm_campaign=designshare&utm_medium=link&utm_source=editor)

![image](https://github.com/user-attachments/assets/65718123-32df-45fe-9895-8fe5029ccf0e)

**Comment by Kexanone** [comment]
Thank you for your proposal! I corrected the title, as it's not about the current ACE Medical, but the prototype test mod.
I don't think we'll directly include the image, especially since the PlantUML format used for the [current activity diagram](https://github.com/acemod/ACE-Anvil/blob/medical-prototypes/cardiovascular-system/extras/models/medical/treatment/advanced/main.svg) is more maintainable. We can discuss how the existing guide can be improved based on this though. For instance, I like that a picture of each item is included, so I propose we'll include such a row in our table.
